### PR TITLE
Task: Add app in docker container

### PR DESCRIPTION
**Pull request by DariaAres**
## Task:
Add app in docker container
## Change proposed in this pull request:
- Created Dockerfile and docker-compose.yml
- Created a rails structure and a database structure to use
- Installed all the gems for the rails application that were pulled in from rails boiler-plate

### CODE REVIEW

**Review by Iorhael — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+    db:
```
Comment (Iorhael) [inline docker-compose.yml:3]: Добавь ещё, пожалуйста, имена контейнерам

File: .env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+MYSQL_ROOT_PASSWORD=DariaAres123!
```
Comment (Iorhael) [inline .env:1]: Для чего здесь дублируются переменные?

File: docker-compose.yml (hunk 1 above)
Comment (DariaAres) [reply docker-compose.yml:3]: добавила

File: .env (hunk 2 above)
Comment (DariaAres) [reply .env:1]: исправила

**Review by Iorhael — APPROVED** [review]

**Review by TsikhanauDzmitry — APPROVED** [review]

**Pull request merged**